README.md

```diff
@@ -223,6 +225,17 @@ But if you want to test or validate the token manually, then send a **POST** req
 }
 ```
 
+## Refreshing the Access Token
+
+For security reasons, third-party applications that are integrating with your authentication server will not store the user's username and password. Instead they will store the refresh token in a user-specific storage that is only accessible for the user. The refresh token can be used to re-authenticate as the same user and generate a new access token.
+
+To generate a new access token, send the refresh token in the HTTP **POST** header _Authorization: Bearer_:
+
+`/wp-json/jwt-auth/v1/token/refresh`
+
+The response is the same as for the `/token` response - only without a new `refreshToken`.
```

Comment:
Some implementations appear to only generate a refresh token in this endpoint, allowing the client to perform a "silent refresh" of the refresh token in order to prolong the user's authentication beyond the default expiry (e.g. 30 days) - so that the user does not have to log in again with username and password. See https://hasura.io/blog/best-practices-of-using-jwt-with-graphql/#refresh_token_expiry

In that case, the responses of the endpoints would change as follows:

1. `/token` generates a new access token. Either by passing username/password combo or a valid refresh token in a HttpOnly cookie.
2. `/token/validate` validates whether a token is still valid. Token can be an access token or refresh token.
3. `/token/refresh` generates a new refresh token as HttpOnly cookie. By passing a valid refresh token sent as cookie.